package-lock.json

```diff
@@ -3127,10 +3127,488 @@
 				"source-map-loader": "^0.2.4",
 				"stylelint": "^9.10.1",
 				"stylelint-config-wordpress": "^13.1.0",
+				"thread-loader": "^2.1.2",
 				"webpack": "4.8.3",
 				"webpack-bundle-analyzer": "^3.0.3",
 				"webpack-cli": "^3.1.2",
 				"webpack-livereload-plugin": "^2.2.0"
+			},
+			"dependencies": {
```

Comment:
This part is surprising and needs to be further investigated. I didn't try it locally but I'm 99% positive that all those dependencies will be installed in `packages/scripts/node_modules` folder.  Ideally, all those dependencies should be reused from the top level `node_modules` folder.

Reply:
Huh, on second glance, I think you'd be correct. It seems to be a recurring pain point the process for adding a new dependency to a package. The workflow I've tried to implement is:

1. _Manually_ add an entry for the new dependency in the specific package
2. From the project root, run `npm install`

Reply:
`npm dedupe` can help here too, though often causes more movement that you'd like.

Reply:
I addressed it in https://github.com/WordPress/gutenberg/pull/14860/commits/ba48f76984fba8d895ba16ab47c87ac97e072fca